cpp/arcticdb/storage/s3/nfs_backed_storage.cpp

```diff
@@ -119,6 +119,39 @@ VariantKey unencode_object_id(const VariantKey& key) {
                         });
 }
 
+NfsBackedStorage::NfsBackedStorage(const LibraryPath &library_path, OpenMode mode, const Config &conf) :
+        Storage(library_path, mode),
+        s3_api_(s3::S3ApiInstance::instance()),  // make sure we have an initialized AWS SDK
```

Comment:
This was missing before and caused this issue when running the tests with a debug build, https://github.com/aws/aws-sdk-cpp/issues/1451